source/ftrack_api/event/hub.py

```diff
@@ -120,6 +120,26 @@ def __init__(self, server_url, api_user, api_key):
             url_parse_result.hostname,
             url_parse_result.port
         )
+        
+        if cookies is not None:
+            if not isinstance(cookies, collections_abc.Mapping):
+                raise TypeError('The cookies argument is required to be a mapping.')
```

Comment:
I am no sure we need to validate the headers and cookies here. Normally the EventHub is initialized from the Session where the headers / and cookies are already validated. I think documenting their expected value should be enough.

We could also remove the "ftrack_strict_api" keyword argument and instead assume that if it is active it is already present in the headers passed in.

Reply:
Done